notebooks/nvidia_haystack_medical_chatbot.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
<p>I dont think many users know about the <code>userdata.get</code> functionality (also, probably not runnable outside of Google Colab). You can replace it with <code>getpass</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>